doc/reference/storage/settings/settings.rst

```diff
@@ -70,6 +70,27 @@ After all data is loaded, the ``h_commit`` handler is issued,
 signalling the application that the settings were successfully
 retrieved.
 
+Storing data to persistent storage
+**********************************
+
+A call to ``settings_save_one()`` uses a backend implementation to store
+settings data to the storage medium. A call to ``settings_save()`` uses an
+``h_export`` implementation to store data bulky using ``settings_save_one()``.
```

Comment:
``store data bulky`` doesn't sound right. Did you mean ``store bulky data``?  But then, I'm not sure what you mean.

Reply:
I wan't to siad that  lot of different data are stored in one "operation"

Reply:
" store data bulky " does not sound right.

Reply:
so, how about saying:
```
A call to ``settings_save()`` uses an
``h_export`` implementation to store different data in one
operation using ``settings_save_one()``.
```

Reply:
@dbkinder Applied your suggestion.